conda/gperftools/info/has_prefix

```diff
@@ -0,0 +1 @@
+/opt/anaconda1anaconda2anaconda3 text etc/activate.d/activate_moose-gperftools.sh
```

Comment:
Can you explain what's going on here?

Reply:
This is a post process operation that alters any paths it sees in the specified file path (`etc/activate.d/activate_moose-gperftools.sh`), having a prefix with all those:
```pre
_placehold_placehold_placehold_placehold*
```
paths, changed to the correct final path of $CONDA_PREFIX. This is so when all is said and done, the end user john_doe who installed Conda to some location, has a working activation script:

```bash
/Users/john_doe/libs/conda3/envs/moose/etc/conda/activate.d/activate_moose-gperftools.sh
export GPERF_DIR=/Users/john_doe/libs/conda3/envs/moose
```
Since we don't know what that will be before hand, this is basically how Conda does it :). I probably should just have linked the following. Will explain better than I can:
Ref: https://docs.conda.io/projects/conda-build/en/latest/resources/make-relocatable.html

Reply:
Looks good - thanks for the explanation!